scenarios/kubernetes_e2e.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Need to figure out why this only fails on travis
+# pylint: disable=bad-continuation
+
+"""Runs kubernetes e2e test with specified config"""
+
+import argparse
+import os
+import subprocess
+import sys
+
+
+def check_output(*cmd):
+    """Log and run the command, return output, raising on errors."""
+    print >>sys.stderr, 'Run:', cmd
+    return subprocess.check_output(cmd)
+
+
+def check(*cmd):
+    """Log and run the command, raising on errors."""
+    print >>sys.stderr, 'Run:', cmd
+    subprocess.check_call(cmd)
+
+
+def main(default_args, job_args):
+    """ set up env, call docker run, clean up """
+    # pylint: disable=too-many-locals
+    # pylint: disable=too-many-branches
+    # pylint: disable=too-many-statements
+
+
+    # platform envs
+    if default_args.platform:
+        platform_env = os.getcwd() + '/platforms/' + default_args.platform + '.env'
+        print 'platform_env : %s' % platform_env
+        output = check_output('cat', platform_env)
+        for line in output.split('\n'):
+            tup = line.strip().split('=', 1)
+            if len(tup) == 2:
+                os.environ[tup[0].strip()] = tup[1].strip()
+
+    # Boilerplate envs
+    # Assume we're upping, testing, and downing a cluster
+    os.environ['E2E_UP'] = 'true'
+    os.environ['E2E_TEST'] = 'true'
+    os.environ['E2E_DOWN'] = 'true'
+
+    e2e_name = 'bootstrap-e2e'
+
+    # Skip gcloud update checking
+    os.environ['CLOUDSDK_COMPONENT_MANAGER_DISABLE_UPDATE_CHECK'] = 'true'
+    # Use default component update behavior
+    os.environ['CLOUDSDK_EXPERIMENTAL_FAST_COMPONENT_UPDATE'] = 'false'
+
+    # AWS variables
+    os.environ['KUBE_AWS_INSTANCE_PREFIX'] = e2e_name
+
+    # GCE variables
+    os.environ['INSTANCE_PREFIX'] = e2e_name
```

Comment:
Previous behavior was export all env as local env var, and import into docker through `--env-files`, which here does the same thing, set all envs and then port into docker.